node/follower_node_test.go

```diff
@@ -204,8 +205,21 @@ func TestSyncRoundWithRemake(t *testing.T) {
 	}
 
 	followNode, _ = remakeableFollowNode(t, tempDir, maxAcctLookback)
-	status, err := followNode.Status()
-	require.NoError(t, err)
+
+	// Wait for follower to catch up. This rarely is needed, but can happen
+	// and cause flakey test failures. Timing out can still occur, but is less
+	// likely than the being caught behind a few rounds.
+	var status *StatusReport
+	for stop := false; !stop; {
```

Comment:
You should be able to use one of these:
* fixture.ClientWaitForRoundWithTimeout(followNode, newRound)
* followNode.WaitForBlock(newRound)
* require.Eventually

Reply:
Using `require.Eventually`.

Reply:
7ab3bb833580c2f59ff5a72f745ee686da6edfd9